recipes/libcurl/all/conanfile.py

```diff
@@ -1,13 +1,20 @@
-import glob
-import os
-import re
-import functools
-from conans import ConanFile, AutoToolsBuildEnvironment, CMake, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
 from conan.tools.microsoft import is_msvc
 from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, chdir, copy, download, get, load, replace_in_file, rm, rmdir, save
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
+from conan.tools.microsoft.subsystems import unix_path
+from conan.tools.scm import Version
+# TODO: Update to conan.tools.apple after 1.51.3
+from conans.tools import is_apple_os, get_env, os_info
```

Comment:
Is it ok to us `from conan.tools.apple import is_apple_os` ? I guess it would imply `required_conan_version = ">=1.51.3"`.